src/module/item/condition/index.ts

```diff
@@ -50,15 +49,6 @@ class ConditionPF2e extends AbstractEffectPF2e {
         systemData.value.value = systemData.value.isValued ? Number(systemData.value.value) || 1 : null;
     }
 
-    /** Set a self roll option for this condition */
-    override prepareActorData(this: Embedded<ConditionPF2e>): void {
-        const slug = this.slug ?? sluggify(this.name);
-        this.actor.rollOptions.all[`self:condition:${slug}`] = true;
-        if (this.slug === "flat-footed") {
-            this.actor.rollOptions.all["self:flatFooted"] = true;
-        }
```

Comment:
Sorry, I meant just get rid of this part :sweat_smile:

Reply:
But its no longer needed, the superclass handles it. The rest of the method only existed to add this part.